website/content/docs/commands/plugin/runtime/register.mdx

```diff
@@ -0,0 +1,55 @@
+---
+layout: docs
+page_title: plugin runtime register - Command
+description: |-
+  The "plugin runtime register" command registers a new plugin runtime in Vault's plugin runtime
+  catalog.
+---
+
+# plugin runtime register
+
+The `plugin runtime register` command registers a new plugin runtime in Vault's plugin runtime catalog.
+Vault currently only supports plugin runtimes of type "container".
+
+To use a registered plugin runtime, use the `-runtime` option with the
+[plugin registration command](/vault/docs/commands/plugin/register).
+
+## Examples
+
+Register a plugin runtime:
+
+```shell-session
+$ vault plugin runtime register -type=container -oci_runtime=runc runc
+Success! Registered plugin runtime: runc
+```
+
+Register a plugin runtime with resource limits:
+
+```shell-session
+vault plugin runtime register \
+    -type=container \
+    -cpu_nanos=100000000 \
+  runsc
+```
+
+## Usage
+
+The following flags are available in addition to the [standard set of
+flags](/vault/docs/commands) included on all commands.
+
+### Command options
+
+- `-type` `(string: <required>)` - Plugin runtime type. Vault currently only
+  supports "container" as a runtime type.
+
+- `-cgroup_parent` `(string: "")` - Parent cgroup to set for each container.
+  This can be used to control the total resource usage for a group of plugins.
+
+- `-cpu_nanos` `(int: 0)` - CPU limit to set per container in nanos. Defaults
```

Comment:
It might be good to clarify here what a "nano" is? I presumed a billionth of a CPU core.

Possibly out the scope of this PR, but maybe it might be more useful to support Kubernetes-like unit suffixes (for here and for memory), so that people could use the more familiar `250m` for a quarter CPU instead of `250000`.

Reply:
Thanks, clarified the language to billionths of a CPU core. And agreed on the unit suffixes - I'm going to try to do that for 1.15.1